05-how-to-contribute-to-open-source.md

```diff
@@ -133,6 +133,30 @@ Replace "your-username" with your GitHub username.
 
 Congratulations! You just made an open source contribution! In the next chapter, we'll add this to an open source portfolio: a resume for open source contributions through which you can showcase your work and attract potential employers and friends using OpenSauced. For now, let's wrap up with some notes on onboarding.
 
+## Understanding Merge Conflicts
```

Comment:
It would make more sense to have the "Resolving Merge Conflict" as part of this section rather than being a `h2`.

Let's do this:


```suggestion
## Merge Conflicts

### Understanding Merge Conflicts
```